akka-projection-core/src/main/scala/akka/projection/ProjectionConfig.scala

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.projection
+
+import scala.concurrent.duration.FiniteDuration
+import scala.concurrent.duration._
+
+import akka.annotation.InternalApi
+import akka.util.JavaDurationConverters._
+import com.typesafe.config.Config
+
+trait ProjectionConfig {
```

Comment:
Still draft, but would like to have early feedback on this. 

That new config should be common to all projections as it defines how to configure the backoff. 

In this impl, I'm passing it to the specific ProjectionImpl, so there are some repetition. Alternatively, this could be an option config for the `ProjectionBehavior` (#104). 

The behavior could then take care of this, but this means that we need to re-think `Projection.run` and `Projection.stop` (which we were planning anyway).

Reply:
I think it belongs to the Projection.

Akka's naming convention would be `ProjectionSettings`